lib/graphql/schema.rb

```diff
@@ -564,6 +564,12 @@ def to_json(*args)
       JSON.pretty_generate(as_json(*args))
     end
 
+    def respects_root_name_conventions?
```

Comment:
Is this concern only for printing the schema? If so, can we keep it out of `schema.rb`? There's already a lot going on in here, and if we can get away with putting it somewhere else, I'd like to do that!

Reply:
done!